packages/wallet-manager/package.json

```diff
@@ -11,16 +11,22 @@
     "dev": "tsc --build --clean && tsc --build ./tsconfig.json --watch",
     "clean": "rm -rf ./build",
     "lint": "eslint 'src/**/*.ts'",
-    "lint:fix": "yarn run lint --fix"
+    "lint:fix": "yarn run lint --fix",
+    "test": "vitest --config ./vite.config.ts"
   },
   "author": "Sygmaprotocol Product Team",
   "devDependencies": {
     "@polkadot/types": "10.7.2",
-    "typescript": "^5.2.2"
+    "jsdom": "^22.1.0",
+    "typescript": "^5.2.2",
+    "vitest": "^0.34.5"
   },
   "dependencies": {
+    "@ethersproject/abstract-signer": "^5.7.0",
+    "@lit/context": "1.0.0",
     "@polkadot/api": "10.7.2",
     "@polkadot/extension-dapp": "^0.46.5",
-    "ethers": "5.7.2"
+    "ethers": "5.7.2",
+    "lit": "3.0.0"
```

Comment:
🤔 I think I missed that when I asked Michael to update the lock file, because I have it in my packae.json file locally. For some reason upon commiting, the pipeline wasn't detecting the changes on the lock, therefore it was failing. I will check this

Reply:
if `widget` uses the same exact version, then it's doesn't matter. but if you update in one package, you'll have two `ethers` versions installed, which would result in a bigger bundle. the other option is to always check that the versions are the same, but IMHO it's less cognitive load to just put peer dep here.

Reply:
we actually use the same version since on versioning we are tied to the version the SDK is using

Reply:
yeah, but what if SDK bump its `ethers` version? then you have to change it in multiple places which is not future-proof (same as magic strings)

Reply:
gotcha

Reply:
🤔 interesting: upon building I needs the `lit/context` as dependency, if not is going to fail